safe_transaction_service/history/tests/test_tx_processor.py

```diff
@@ -38,7 +40,7 @@
 logger = logging.getLogger(__name__)
 
 
-class TestSafeTxProcessor(TestCase):
+class TestSafeTxProcessor(SafeTestCaseMixin, TestCase):
```

Comment:
Is this to perform an initial setup for the test case?

Reply:
Yes, and extends this class to deploy the safe contract. `self.deploy_test_safe()`